search-ui/src/components/Transcript/Transcript.js

```diff
@@ -25,58 +37,37 @@ export default function Transcript(props) {
         return null;
     }
 
-    const overflow = {
-        overflowX: 'auto'
-    };
-
-    const halfWidth = {
-        width: '50%'
-    };
-
-    console.log(full_content);
-
-    if (!!props.document.translated_text && props.document.translated_text !== null && props.document.language !== "en") {
-        return (
-            <div style={overflow}>
-                <table className="table table-hover table-striped table-bordered">
-                    <thead>
-                        <tr>
-                            <th>Original Content</th>
-                            <th>Translated (En)</th>
-                        </tr>
-                    </thead>
-                    <tbody>
-                        <tr>
-                            <td className="wrapword" style={halfWidth}>
-                                <pre id="transcript-viewer-pre">{full_content}</pre>
-                            </td>
-                            <td className="wrapword">
-                                <pre>{props.document.translated_text.trim()}</pre>
-                            </td>
-                        </tr>
-                    </tbody>
-                </table>
-            </div>
-        );
-    }
-    else {
-        return (
-            <div style={overflow}>
-                <table className="table table-hover table-striped table-bordered">
-                    <thead>
-                        <tr>
-                            <th>Original Content</th>
-                        </tr>
-                    </thead>
-                    <tbody>
-                        <tr>
-                            <td className="wrapword">
-                                <pre id="transcript-viewer-pre">{full_content}</pre>
-                            </td>
-                        </tr>
-                    </tbody>
-                </table>
-            </div>
-        );
-    }
+    function GetReferences(searchText, content) {
+        // find all matches in content
```

Comment:
Perhaps give a little more info in this comment:  
// find all matches in content and insert html that will add highlighting and a reference/id

Reply:
good point; comment added